usr.bin/ncal/ncal.c

```diff
@@ -345,6 +347,9 @@ main(int argc, char *argv[])
 		case 'y':
 			flag_wholeyear = 1;
 			break;
+		case 'M':
+			flag_monday = 1;
+			break;
```

Comment:
350,352m310

Ok, I guess I can comment *a little* on the src. Keeping things alphabetized is an important part of maintainability.

But, my approval only counts for manuals at this time.

Reply:
ok moved